sw/device/tests/crypto/cryptotest/BUILD

```diff
@@ -0,0 +1,122 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+load(
+    "//rules/opentitan:defs.bzl",
+    "cw310_params",
+    "opentitan_binary",
+    "opentitan_test",
+)
+
+package(default_visibility = ["//visibility:public"])
+
+# TODO: There is still a bug that prevents running all the tests in one go.
+# For now, run the tests separately
```

Comment:
Is this bug still active? If so, it might be good to create an issue and reference it here so it's tracked.

Reply:
Unfortunately yeah, it's still a problem. I'm still looking into it now which is why I left this comment in here, but if I don't get anywhere, I'll open an issue and clean this up.

Reply:
Just so the folks here get a ping, I'll note that in https://github.com/lowRISC/opentitan/issues/20277, it actually appears that the AES block and firmware aren't happy with each other. It appears AES is stuck waiting for input, and the firmware is spinning as it waits for AES to go idle.